examples/dreambooth/README.md

```diff
@@ -737,3 +737,7 @@ accelerate launch train_dreambooth.py \
   --class_labels_conditioning timesteps \
   --push_to_hub
 ```
+
+## Stable Diffusion XL
+
+We support fine-tuning of the UNet shipped in [Stable Diffusion XL](https://github.com/Stability-AI/generative-models/blob/main/assets/sdxl_report.pdf) with DreamBooth and LoRA via the `train_dreambooth_lora_sdxl.py` script. Please refer to the docs [here](./README_sdxl.md). 
```

Comment:
Might be better to link the HuggingFace page for SDXL's report, or the arxiv page, rather than the GitHub copy.

Reply:
Thanks for suggestion! Rectified.